op-deployer/pkg/deployer/inspect/deploy_config.go

```diff
@@ -42,5 +43,10 @@ func DeployConfigCLI(cliCtx *cli.Context) error {
 		return fmt.Errorf("failed to write deploy config: %w", err)
 	}
 
+	chainState.Artifacts.DeployConfig = filepath.Join(cliCfg.Workdir, cliCfg.Outfile)
```

Comment:
Inspect commands should never modify state. What are you trying to do here?

Reply:
I am trying to automate the process as much as possible so there is less onus on the user to collect all this information manually. Since we already have inspect commands that create the artifacts needed by the superchain-registry, I figured storing the filepaths at the time of creation would save the user some work down the road. Can put some checks in that would error if the file has been moved.

Maybe I should back up and ask: what is the purpose of the `inspect superchain-registry` command? Do we just want to automate the following fields:
```
SCR_GENESIS_CREATION_COMMIT
SCR_SUPERCHAIN_TARGET
```
Then have the user manually fill out the rest (via cli flags?):
```
SCR_GENESIS
SCR_STANDARD_CHAIN_CANDIDATE
SCR_CHAIN_NAME
SCR_CHAIN_SHORT_NAME
SCR_SEQUENCER_RPC
SCR_DEPLOYMENTS_DIR
SCR_PUBLIC_RPC
SCR_EXPLORER
SCR_ROLLUP_CONFIG
SCR_DEPLOY_CONFIG
```